.woodpecker/docker.yml

```diff
@@ -68,13 +68,38 @@ pipeline:
       branch: ${CI_REPO_DEFAULT_BRANCH}
       event: push
 
+  publish-branch-server:
+    image: plugins/docker
+    group: docker
+    secrets: [docker_username, docker_password]
+    settings:
+      repo: woodpeckerci/woodpecker-server
+      dockerfile: docker/Dockerfile.server
+      tag: ${CI_COMMIT_BRANCH##release/}
+    when:
+      branch: release/*
+      event: push
+
+  publish-branch-server-alpine:
+    image: plugins/docker
+    group: docker
+    secrets: [ docker_username, docker_password ]
+    settings:
+      repo: woodpeckerci/woodpecker-server
+      dockerfile: docker/Dockerfile.server.alpine
+      tag: ${CI_COMMIT_BRANCH##release/}
+    when:
+      branch: release/*
+      event: push
+
   release-server:
     group: docker
     image: plugins/docker
     secrets: [docker_username, docker_password]
     settings:
       repo: woodpeckerci/woodpecker-server
       dockerfile: docker/Dockerfile.server
+      # remove 'latest' on older version branches to avoid accidental downgrade
```

Comment:
This would happen if  we would tag on a release branch, right?

Reply:
yes but if there are more release branches lets say 'release/v0.15' & 'release/v0.16' and we would tag for example 'v0.15.3' after 'v0.16.2' we would have a downgrade on latest.

this is because we can not determine a branch for a tag event else i could add a when.branch filter too it ...

so we have to remove 'latest' tag on pipeline config on all release branches who are not latest release branch ...